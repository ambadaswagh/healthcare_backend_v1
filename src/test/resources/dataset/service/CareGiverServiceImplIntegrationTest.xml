<?xml version="1.0" encoding="UTF-8"?>
<dataset>

    <company id="100"
             license_no="License"
             federal_tax="Federal Tax"
             federal_tax_start="2017-05-01"
             federal_tax_expire="2017-05-20"
             federal_tax_status="1"
             state_tax="State Tax"
             state_tax_start="2017-05-01"
             state_tax_expire="2017-05-20"
             state_tax_status="1"
             name="Name"
             phone="12345"
             fax="12345"
             email="test@email.com"
             address_one="Address 1"
             address_two="Address 2"
             city="City"
             state="State"
             zipcode="00000"
             status="1" />

    <agency_type id="100"
                 name="Name"
                 status="1" />

    <agency id="100"
            license_no="License"
            company_id="100"
            company_id1="100"
            agency_type_id="100"
            name="Name"
            tracking_mode="1"
            contact_person="Contact Person"
            email="test@email.com"
            address_one="Address 1"
            address_two="Address 2"
            city="City"
            state="State"
            zipcode="00000"
            timezone="UTC"
            phone="12345"
            holiday="Holiday"
            fax="12345"
			status="1"/>

		 <caregiver id = "100"
		  address_one= "Address 1"
		  address_two= "Address 2"
		  address_type= "Permanent"
		  caregiver_type= "1"
		  certificate= "My Certificate"
		  certificate_end= "2017-04-20"
		  certificate_start= "2017-05-01"
		  city= "city"
		  created_at= "2017-04-20"
		  dob= "2017-06-24"
		  email= "test@email.com"
		  first_name= "first name"
		  gender= "Male"
		  language= "English"
		  last_name= "last name"
		  middle_name= "middleName"
		  phone= "123456789"
		  secondary_phone= "secondaryPhone"
		  social_security_number= "12345"
		  state= "state"
		  status= "1"
		  updated_at= "2017-04-20"
		  username= "testuser1"
		  password = "testpassword"
		  vacation_end= "2017-05-20"
		  vacation_note= "vacation notes"
		  vacation_start= "2017-04-20"
		  verification_code= "Hard Copy"
		  zipcode= "123456"
		  company_id="100"
		  agency_id="100"/> 
</dataset>