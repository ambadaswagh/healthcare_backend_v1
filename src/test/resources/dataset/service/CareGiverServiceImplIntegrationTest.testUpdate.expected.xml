<?xml version="1.0" encoding="UTF-8"?>
<dataset>
		 <caregiver id = "100"
		  first_name = "first name updated"
		  company_id="100"
		  agency_id="100"/> 
</dataset>